packages/workbox-webpack-plugin/src/index.js

```diff
@@ -115,34 +115,39 @@ class WorkboxWebpackPlugin {
      * compiler.plugin('watch-run') for development.
      */
     setReadFile(compiler.inputFileSystem._readFile);
+
     /**
-     * During the make phase of the webpack compilation, we use
+     * During the make phase of the Webpack compilation, we use
```

Comment:
I think webpack is meant to be all lower case as weird as it may seem.

From: http://webpack.github.io/docs/

> If you are new to webpack, you may want

Reply:
Go figure.